### PR TITLE
Fix inline config example in README

### PR DESCRIPTION
**Pull request by csabapalfi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ Or use the plugin API to do things like:
 gulp.src(['**/*.js','!node_modules/**'])
 	.pipe(eslint({
 		extends: 'eslint:recommended',
-		ecmaFeatures: {
-		    'modules': true
+		parserOptions: {
+		    'sourceType': 'module',
```
Comment (shinnn) [inline README.md]: No comma here.

**Comment by shinnn** [comment]
Good catch, @csabapalfi! Just added one comment. https://github.com/adametry/gulp-eslint/pull/159#discussion_r68773032

File: README.md (hunk 1 above)
Comment (shinnn) [inline README.md]: Also `sourceType` doesn't need to be quoted.

```js
		    sourceType: 'module'
```

**Comment by shinnn** [comment]
https://github.com/adametry/gulp-eslint/pull/159#discussion_r68774735

**Comment by csabapalfi** [comment]
Thanks for the quick reply. Fixed everything now I think. Also reverted to tabs (as I accidentally switched to spaces first)

**Comment by shinnn** [comment]
Perfect. Thanks.

**Pull request merged**